docs/supporting/faq.md

```diff
@@ -115,7 +115,7 @@ In conclusion, we expect our approach to evolve as we learn more about GDPR and
 
 You can find more information about how the Visual Studio family approaches GDPR at [Visual Studio Family Data Subject Requests for the GDPR](https://docs.microsoft.com/en-us/microsoft-365/compliance/gdpr-dsr-visual-studio-family).
 
-## How to disable telemetry reporting
+## How do I disable telemetry reporting?
```

Comment:
We shouldn't change existing H2 headers as this will break existing links to sections.

Reply:
Reverted.